.travis.yml

```diff
@@ -1,27 +1,127 @@
+######## Testing minimal compiler requirements ########
+# GCC          6.1.0        here: 6.4.0
+# Clang        4.0          here: 3.8,3.9 not compiling
```

Comment:
Does this mean that we are testing clang 4, and that 3.8 & 3.9 don't work?
And does the comment above say that we require GCC 6.1? and here we are testing GCC 6.4?

Reply:
OK. So, I would propose removing any mention of 3.8 and 3.9 from here, and updating the installation guide to not mention Clang 3.x.

Reply:
Will do. Where do I update the install guide?

Reply:
You can update the documentation in this PR.
The docs are generated from files in the `doc/` path, in this case `doc/install.rst`:
https://github.com/eth-cscs/arbor/blob/master/doc/install.rst
Just edit this in your text editor.

Reply:
Do you want me to update gcc 6.1 to 6.4 as well as the note beneath where we still use 5.2.0?

Reply:
No. I have tested it with gcc 6.1 on Daint.
We have to support 6.1 at least.

Reply:
And yes, please move 5.2.0 to 6.1 wherever you see it!

Reply:
Ok, what about the gcc5.2.0 note? update to 6.1.0?

Reply:
ok

Reply:
Does the Intel compiler note need to be updated as well from "Needs GCC 5 or later for standard library" to "Needs GCC 6 ..."?

Further, for Travis I needed gcc6 for clang on linux as well for std lib. Shall I put that in the note for clang?